src/ctia/store.clj

```diff
@@ -2,13 +2,13 @@
 
 (defprotocol IActorStore
   (read-actor [this id])
-  (create-actor [this new-actor])
+  (create-actor [this new-actors])
```

Comment:
I think I would prefer that we keep the existing create-<entity> and add a create-<entity>s